frontend/src/components/details/person_media.svelte

```diff
@@ -1,56 +1,64 @@
 <script lang="ts">
-  import { mediaIdToUrlConverter } from "../../utils"
+  import InfiniteLoading from "svelte-infinite-loading"
+  import {
+    mediaIdToUrlConverter,
+    calculateAmountOfShownItems,
+    removeDuplicates,
+    sortListByPopularity,
+    removeContentWithMissingImagePath,
+  } from "../../utils"
 
   const LOW_RES_IMG_URL: string = "https://image.tmdb.org/t/p/w500/"
-  const SHOW_BUTTON_AMOUNT: number = 12
-  const CAST_ITEM_START_AMOUNT: number = 6
-
   export let media: []
-  export let mediaType: string
-  export let title: string
+  let currentMediaAmount = calculateAmountOfShownItems(
+    window.visualViewport.width,
+    [32, 28, 24, 20, 16, 10]
+  )
+  const mediaStartAmount = currentMediaAmount
+
+  const loadMoreData = async ({ detail: { loaded } }) => {
+    currentMediaAmount += mediaStartAmount
+    loaded()
+  }
+
+  const getMediaTitle = media => {
+    if (media.id.charAt(0) == "m") {
+      return media.title
+    } else {
+      return media.name
+    }
+  }
 
-  let mediaAmount: number = 6
+  removeDuplicates(media)
+  removeContentWithMissingImagePath(media, "poster_path")
+  sortListByPopularity(media)
 </script>
 
 {#if media.length}
-  <h1 class="text-center text-3xl pt-5">{title}</h1>
+  <h1 class="text-center text-3xl pt-5">Movies & Series</h1>
```

Comment:
Might be better with a "Starred in" or something that makes it obvious that the person is a part of them

Reply:
Done